omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
Depends on how you declare loadTags() and when you call it, are you calling as the last line in init()? And are you declaring it as a scoped function or as a member of the main function you pass to the module?